lib/modules.nix

```diff
@@ -1256,7 +1256,7 @@ let
       (opt.highestPrio or defaultOverridePriority)
       (f opt.value);
 
-  doRename = { from, to, visible, warn, use, withPriority ? true }:
+  doRename = { from, to, visible, warn, use, withPriority ? true, condition ? true }:
```

Comment:
We don't have any docs for this function, but at least an inline comment here would be good

Reply:
Thanks! Here's the follow-up:
- https://github.com/NixOS/nixpkgs/pull/286544